### DISCUSSION TITLE
Silly but necessary question of auto updating

### DISCUSSION THREAD

**Discussion by DeltaPheonix-Dev** [community/community#84870] [New to GitHub] [answered]
I have VSCode on my Windows PC signed in and synced to my Github Dev page.  I have recently been using and updating my current project via the VSCode.dev (Browser version of VSCode).
I have pushed and committed changes but the project on my PC version of VSCode is not updating with the changes I have made via the browser version.
Am I missing something?  I wish to work via both and have them automatically update aka if I work on the browser version on my ipad, when I start my PC version of VSCode up, it will automatically update and visa versa.

**Comment by DeltaPheonix-Dev** [comment, marked as answer]
I have just searched online and found out that I need to fetch the updates via my source control on VSCode windows.

**Reply by queenofcorgis** [reply]
Thanks for coming back and sharing the solution @DeltaPheonix-Dev 🎉
